templates/form-fields/file-field.php

```diff
@@ -4,7 +4,7 @@
 $field_name         = isset( $field['name'] ) ? $field['name'] : $key;
 $field_name         .= ! empty( $field['multiple'] ) ? '[]' : '';
 
-if ( ! empty( $field['ajax'] ) ) {
+if ( ! empty( $field['ajax'] ) && job_manager_user_can_upload_file_via_ajax() ) {
```

Comment:
That works. Since Ajax uploads would technically be allowed in previous versions, you could also just do a WPJM version check. Either works, though!